crates/ruff_linter/src/rules/pycodestyle/rules/literal_comparisons.rs

```diff
@@ -48,7 +48,22 @@ impl EqCmpOp {
 ///     pass
 /// ```
 ///
+/// ## Fix safety
+///
+/// This rule's fix is marked as unsafe, as it may alter runtime behavior when
+/// used with libraries that override the `==`/`__eq__` or `!=`/`__ne__` operators.
+/// In these cases, `is`/`is not` may not be equivalent to `==`/`!=`.
+///
+/// Notable examples and alternatives include:
+/// - `pandas.DataFrame`, often used with `DataFrame.mask`, `DataFrame.where` - see [pandas.DataFrame.eq].
+/// - `pandas.Series`, often used with `Series.mask`, `Series.where` - see [pandas.Series.eq].
+/// - `sqlalchemy.Column`, often used with `Query.having`, `Query.filter`, `Query.where` - see [sqlalchemy.sql.expression.ColumnOperators.is_].
+/// - `numpy.Array`, often used with `Array.where` - no alternative.
```

Comment:
Unsure if this level of detail is appropriate. Perhaps a link to the relevant issue (#4560) is preferred.

Reply:
Yeah, I think the first paragraph describing the general reason why this is an unsafe fix is good, but not sure we need to go into all these examples. An inline link to the issue sounds like a good idea!